### PR TITLE
Put some basic tree seqs & sample files into fixtures

### PR DESCRIPTION
**Pull request by hyanwong**
This saves some code and is useful for future testing

### CODE REVIEW

**Comment by hyanwong** [comment]
I guess @benjeffery would be best placed to review this, as it's a pytest thing.

File: tests/conftest.py [hunk 1]
```diff
@@ -59,3 +65,48 @@ def pytest_collection_modifyitems(config, items):
...
+    """
+    A simple 1-tree sequence with at least 2 inference sites
+    (i.e. mutations above a non-sample node)
+    """
+    ts = msprime.simulate(10, mutation_rate=1, random_seed=1)
+    assert num_nonsample_muts(ts) > 1
+    return ts
+
+
+@fixture(scope="session")
```
Comment (benjeffery) [inline tests/conftest.py:85]: Is `SampleData` immutable? Otherwise you can't set `scope` to session.

**Review by benjeffery — APPROVED** [review]
LGTM - just one question about scope.

File: tests/conftest.py (hunk 1 above)
Comment (hyanwong) [reply tests/conftest.py:85]: Thanks Ben. You can't modify a SampleData file - you have to take a copy. There's a flag ".READ_MODE" on it that is set once it is made, and all write methods check this before doing anything.  ".READ_MODE" can't be unset AFAIK. So I think the answer is that practically, it is immutable, although perhaps that's not known by Python.

**Comment by benjeffery** [comment]
@jeromekelleher not sure what out merging etiquette is for `tsinfer` yet, but this LGTM!

**Review by jeromekelleher — APPROVED** [review]
LGTM, nice to get rid of some of this code.

**Pull request merged**